### ISSUE TITLE
Accessibility: bottom app list not usable with talkback

### ISSUE THREAD

**Issue by davidbtc2009** [SimpleMobileTools/Simple-Launcher#15]
The bottom row of icons when on the home screen cannot be focused when using the talkback screen reader
